### PR TITLE
Bring FitBoxes to front on click.

### PR DESCRIPTION
**Pull request by papr**
Fixes https://github.com/pupil-labs/pupil/issues/363

@cpicanco could you test this branch and confirm functionality?

### CODE REVIEW

File: pyglui/ui.pyx [hunk 1]
```diff
@@ -598,6 +613,9 @@ cdef class FitBox:
         def __get__(self):
             return self.org.x+self.size.x/2,self.org.y+self.size.y/2
 
+    def py_mouse_over(self, Vec2 m):
+        return self.mouse_over(m)
+
     cdef bint mouse_over(self,Vec2 m):
```
Comment (mkassner) [inline pyglui/ui.pyx]: make this a cpdef and you can get rid of the pywrapper.

**Comment by mkassner** [comment]
This could actually work.( I have not tested it yet)  great work @papr  . I made a comment in the commit . please have a look.

**Comment by papr** [comment]
Done. It works so far.

**Comment by cpicanco** [comment]
Will test!

**Comment by papr** [comment]
@cpicanco Let us try to keep the main discussion here. You can reference the PRs and I will merge them directly for testing.

Just for redundancy: Did you reinstall `pyglui`?

**Comment by cpicanco** [comment]
Perfect! It is working!

**Comment by papr** [comment]
I suggest reverting the re-ordering functionality since we do not use this kind of menus anymore and the reordering is producing visualization errors in the new version of Pupil.

**Pull request merged**